cos-gpu-installer-docker/entrypoint.sh

```diff
@@ -394,7 +440,8 @@ main() {
   else
     info "Did not find cached version, building the drivers..."
     download_kernel_src
-    install_cross_toolchain_pkg
+    local -r download_url=set_compilation_env
```

Comment:
This should be "$(set_compilation_env)". Otherwise it means download_url="set_compilation_env"

Reply:
Also s/download_url/toolchain_download_url to make it clear it is for toolchain tarbar.